pkg/kratos/interfaces.go

```diff
@@ -29,6 +29,8 @@ type AuthorizerInterface interface {
 type ServiceInterface interface {
 	CheckSession(context.Context, []*http.Cookie) (*kClient.Session, []*http.Cookie, error)
 	AcceptLoginRequest(context.Context, string, string) (*hClient.OAuth2RedirectTo, []*http.Cookie, error)
+	GetLoginRequest(context.Context, string) (*hClient.OAuth2LoginRequest, []*http.Cookie, error)
```

Comment:
food for thought: this method seems to only be used in the `MustReAuthenticate`, so maybe we could avoid exposing it?
You already wrote tests for it so I'm happy keeping it here

Reply:
Makes sense, I will remove it from the interface